src/main/java/com/untamedears/humbug/Versioned.java

```diff
@@ -1,8 +1,9 @@
 package com.untamedears.humbug;
 
-import org.bukkit.craftbukkit.v1_10_R1.entity.CraftHorse;
-import net.minecraft.server.v1_10_R1.EntityHorse;
-import net.minecraft.server.v1_10_R1.GenericAttributes;
+import net.minecraft.server.v1_11_R1.EntityHorse;
+import net.minecraft.server.v1_11_R1.GenericAttributes;
+
+import org.bukkit.craftbukkit.v1_11_R1.entity.CraftHorse;
 import org.bukkit.entity.Entity;
 
 public class Versioned {
```

Comment:
I'm surprised it's still b() on the AttributeInstance, but it's good if that API has remained stable under the hood.